extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -146,17 +147,9 @@ export class SchemaCompareMainWindow {
 
 				this.splitView = this.view.modelBuilder.splitViewContainer().component();
 
-				let sourceTargetLabels = this.view.modelBuilder.flexContainer()
-					.withProperties({
```

Comment:
are these property values the same as the default values?

Reply:
Not sure I understand - but I'm removing them because there are no properties for alignItems or horizontal. Those are layout properties and so putting them in the properties was not doing anything. 

A follow up question would be "should these be moved to withLayout instead" - but that should be done separately - I'm trying to avoid doing any changes here that actually impact anything.

Reply:
does the schema compare window look the same after you removed these 2 properties? If so, I think these can be removed, otherwise, we should add those to the correct place asap so that we don't leave the extension in a bad state.

Reply:
![image](https://user-images.githubusercontent.com/28519865/123161092-d8224300-d423-11eb-90c6-83fa7acfb020.png)

Reply:
@kisantia , looks like these 2 properties doesn't have any effect, looks good to me to remove them. please confirm.

Reply:
looks good to me